event_orchestration.go

```diff
@@ -0,0 +1,577 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/google/go-querystring/query"
+)
+
+const (
+	eoPath = "/event_orchestrations"
+)
+
+// Orchestration defines a global orchestration to route events the same source
+// to different services.
+type Orchestration struct {
```

Comment:
Could we name the types and methods in this file `EventOrchestration`, `ListOrchestrations`, etc.

Reply:
Actually maybe let me look at this a bit further. Now I see why you did this.

Reply:
Okay now I understand why you did this. We have some discrepancy between the API path (`/event_orchestrations`) and the objects themselves (`orchestrations`). I like what you did here.